packages/openapi-typescript/test/transform/operation-object.test.ts

```diff
@@ -155,6 +155,65 @@ responses: {
             "application/json": string;
         };
     };
+};`,
+      },
+    ],
+    [
+      "defaultNonNullable > parameters aren’t required even with defaults",
```

Comment:
This new test asserts that a `parameter` with a default is ignored, but a response object with a default is not